gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributesTest.java

```diff
@@ -29,14 +31,13 @@
 
   private static final byte[] HAPPY = "(✿◕ ‿◕ )ノ".getBytes(UTF_8);
 
-  @Rule
-  public final AppEngineRule appEngineRule = new AppEngineRule();
 
   private Path path;
   private Path dir;
 
   @Before
   public void before() throws Exception {
+    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
```

Comment:
I disagree with this one. When writing tests, it is my preference to put `throws Exception` on everything, since it reduces cognitive burden. There's no concrete benefit to documenting exceptions using `throws` in test code, because `thrown.expect` methods do that.